### ISSUE TITLE
Tried add entity_add functionality but not working.

### ISSUE THREAD

**Issue by ameygat** [yeti-platform/pyeti#5]
I am trying to add more functionality to the YetiApi class.
When I tried the get entity it worked, but for adding a entity it gives error

ERROR:root:An error occurred (500):

Any pointers towards this would be welcome, There was no documentation of json values of entity object so I did a get entity and used similar json object in entity_add

Following are my functions added to api.py
```
    def entity_details(self, id):
        """Get details on an entity.
        Args:
            id: A string representing the entity's ObjectID

        Returns:
            JSON representation of the requested entity
        """
        return self._make_get("entity/{}".format(id))

    def entity_add(self, name,etype,description,aliases=[], tags=[],context={},source="API"):
        """Add an entity to the dataset

        Args:
            name: the name of entity
            etype: type of entity ('Actor','TTP' etc)
            description: description of Entity
            tags: An array of strings representing tags
            context: A dictionary object with context information
            source: A string representing the source of the data. Defaults to
                   "API".

        Returns:
            JSON representation of the created observable.
        """
        json = {
            'description':description,
            'tags':tags,
            'aliases':aliases,
            'type':etype,
            'name':name,
            'context':context,
            'source':source
        }

        return self._make_post('entity/', json=json)
```

**Comment by tomchop** [comment]
Have you tried running the webserver in dev mode? This will give you a full stack trace and it will be much easier to narrow down the error.

**Comment by ameygat** [comment]
No I have not tried that, How can I do that could you kindly point me to any link / doc /information about this.

**Comment by ameygat** [comment]
@tomchop Any thing from your side ? Can put some light on this matter ?

**Comment by tomchop** [comment]
Right, so it according to this line: `FieldDoesNotExist: The fields "set([u'source', u'type', u'context', u'aliases'])" do not exist on the document "Entity"` It seems that you're trying to add an Entity with those fields, which does not match the model defined in Yeti.

`POST`ing to `/entity` is the right(ish—see below) way to go, but you need a JSON object that matches what Yeti expects. I know that the documentation is very light on what is expected, but the best way to know is to create an entity via the UI and then `GET` the JSON representation of it.

Also, it seems that you're creating an `Entity` object when you actually want sub-objects (`Actor`, `Malware`, etc.). You'll have to manually set the `_cls` attribute in your request. Also, you'll have to set fields according to [entity.py](https://github.com/yeti-platform/yeti/blob/master/core/entities/entity.py) and [actor.py](https://github.com/yeti-platform/yeti/blob/master/core/entities/actor.py) (if what you want to add is an actor).

**Comment by Kirtar22** [comment]
@ameygat  - did you manage to get the Entity added to Yeti? for me, even adding entity with minimal parameters also not working. 

ERROR:root:An error occurred (500): http://localhost:5000/api/entity/
